lib/rules/check-hbs-template-literals.js

```diff
@@ -75,7 +75,7 @@ module.exports = {
           if (node.quasi.type === 'TemplateLiteral') {
             let hbs = node.quasi.quasis[0].value.cooked
             hbs = unindentAndStripSafeNewlines(hbs)
-            const results = linter.verify({source: hbs.toString(), moduleId: context.id})
+            const results = linter.verify({source: hbs.toString(), filePath: context.id, moduleId: context.id})
```

Comment:
```suggestion
            const results = linter.verify({source: hbs.toString(), filePath: context.id, moduleId: context.getFilename()})
```

based on [eslint docs](https://eslint.org/docs/4.0.0/developer-guide/working-with-rules#the-context-object), `context.id` seems like the wrong thing here.

Reply:
It's a shame this was merged without this fix; I just hit #40 as well and came to the same conclusion—`context.id` is just the rule name `check-hbs-template-literals`, which doesn't really make sense as the `filePath` or `moduleId`